frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/AppearanceSettings.tsx

```diff
@@ -232,6 +232,7 @@ export const AppearanceSettings = ({
               secretKey={secretKey}
               params={params}
               displayOptions={displayOptions}
+              embedModalLocation={EMBED_MODAL_TABS.Appearance}
```

Comment:
I needed to add the location since each tab has its own instance of the same `CodeSample` component with the static embedding code, so the snowplow event can track that

Reply:
Ok, sounds good. Maybe we need to get that PR in before this one.